WalletWasabi.Fluent/Controls/Dialog.axaml.cs

```diff
@@ -108,10 +108,7 @@ private void UpdateDelay(bool isDialogOpen)
 				{
 					CancelPointerPressedDelay = new CancellationTokenSource();
 
-					Task.Delay(TimeSpan.FromSeconds(1), CancelPointerPressedDelay.Token).ContinueWith(_ =>
-					{
-						_canCancelOnPointerPressed = true;
-					});
+					Task.Delay(TimeSpan.FromSeconds(1), CancelPointerPressedDelay.Token).ContinueWith(_ => _canCancelOnPointerPressed = true);
```

Comment:
I assume this kind of _fire and forget_ pattern is common in ui but in the rest of the code base is not so, that's why it seems unusual to me. I would prefer to avoid continuations in this way and use await, i mean, encapsulate this in a helper or something like a 'FireAndForget(()=>{await delay; action())'.

This is just a suggestion, it is okay as it is.

Reply:
I agree, but we will refactor it when we will do the "great fluent review".